packages/vue-app/template/client.js

```diff
@@ -81,8 +81,10 @@ const errorHandler = Vue.config.errorHandler || console.error
 createApp()
   .then(mountApp)
   .catch((err) => {
-    err.message = '[nuxt] Error while mounting app: ' + err.message
-    errorHandler(err)
+    const improvedErr = new Error('[nuxt] Error while mounting app: ' + err.message)
+    improvedErr.original = err
+    improvedErr.stack = improvedErr.stack.split('\n').slice(0,2).join('\n') + '\n' + err.stack
```

Comment:
This is a very low-level hack which may make error handler even more unstable for different browsers / errors

Reply:
Could be, yeah. But What would be another solution? Checking for a DOMException (and other errors with read-only values)? :thinking:

Reply:
Another thing is that in IE for example improvedErr.stack would be undefined and crash
```
new Error().stack
undefined
```